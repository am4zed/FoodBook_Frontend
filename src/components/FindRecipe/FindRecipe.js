import React from 'react';
import Searchbar from '../Searchbar/Searchbar';
import './style.css';

const FindRecipe = () => {
  return(
    <>
      <h1>Welcome to Foodbook.</h1>;
      <h3>Search for recipes that include your ingredient.</h3>;
      <Searchbar />
    </>
  )
}

export default FindRecipe;
