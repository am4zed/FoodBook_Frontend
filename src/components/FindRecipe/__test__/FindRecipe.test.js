import React from "react";
import SearchBar from "../../Searchbar/Searchbar.js"
import FindRecipe from "../FindRecipe.js"
import { shallow } from 'enzyme';

describe("FindRecipe", () => {

    let wrapper;

    beforeEach(() => wrapper = shallow(<FindRecipe />));
    // TESTS FAILING
    it("renders a <Searchbar>", () => {
        expect(wrapper.containsMatchingElement(<Searchbar></Searchbar>).toEqual(true));
    })

    it("renders without crashing", () => {
        expect(wrapper).toMatchSnapshot();
    })
    
})