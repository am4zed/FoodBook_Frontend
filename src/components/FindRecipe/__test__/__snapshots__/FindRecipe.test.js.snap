// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FindRecipe renders without crashing 1`] = `undefined`;
